quarkus/all-service/src/main/resources/META-INF/openapi.yaml

```diff
@@ -10,15 +10,32 @@ paths:
     post:
       requestBody:
         content:
-          application/json: {}
+          application/json:
+            schema:
+              $ref: '#/components/schemas/User'
+            examples:
+              new-ninja:
```

Comment:
Shouldnt the example schemas be in JSON format?

Reply:
The format of the message is defined as json. It is shown in the same format as the document. Unless we change the whole spec to json, it won't be shown in json.